view/homeView.php

```diff
@@ -0,0 +1,10 @@
+<?php
+ob_start();
+?>
+
+<h1>Hello!</h1>
+
+<?php
+$content = ob_get_clean();
+require('template.php');
+?>
```

Comment:
if you have twig lib in your composer , you should use twig to render all views.